src/Avalonia.DotNetFrameworkRuntime/RuntimeInfo.cs

```diff
@@ -20,6 +28,7 @@ internal partial class StandardRuntimePlatform
                 IsMobile = false,
                 IsUnix = isUnix,
                 OperatingSystem = isUnix ? DetectUnix() : OperatingSystemType.WinNT,
+                IsDirectXCompositionAvailable = isDirectXCompositionAvailable
```

Comment:
DirectX is a single-platform thing, so it's probably better to keep that detection in desktop-specific AppBuilder anyway.